pkg/splunk/enterprise/configuration.go

```diff
@@ -639,7 +640,29 @@ func updateSplunkPodTemplateWithConfig(client splcommon.ControllerClient, podTem
 		if spec.ClusterMasterRef.Namespace != "" {
 			clusterMasterURL = splcommon.GetServiceFQDN(spec.ClusterMasterRef.Namespace, clusterMasterURL)
 		}
+		//Check if CM is connected to a LicenseMaster
```

Comment:
Does this also work with [External License Master](https://splunk.github.io/splunk-operator/Examples.html#using-an-external-license-master)?

Reply:
TBH, I haven't tried with that. This PR just focusses on LM created with splunk operator as of now.

Reply:
Ok, in case we find it not working, I suggest we create a separate bug which can then be picked up once CSPL-507 is fixed